docs/intro/ff-ui/toolbar.md

```diff
@@ -0,0 +1,234 @@
+
+:::info
+
+To tag users, select the @ symbol and choose the project team member(s).
+
+:::
+
+## 8. Optimizations & Enhancements
+
+We scan and suggest enhancements to elevate your app's design and speed. Imagine having a personal consultant for design and performance dedicated to improving the user experience of your app.
+
+Optimizations: We spotlight elements that may slow down your app. For example, a query on a column, unused and duplicate backend query, a widget with unbounded size, etc.
+
+UI Enhancements: We offer tips for a more visually appealing and user-friendly design, like resizing a small tap target of any widget.
+
+:::info
+
+You can control what kind of suggestions you would like to receive by clicking on the settings icon on the right.
+
+:::
+
+![Optimization.avif](imgs/Optimization.avif)
+
+## 9. Project issues
+
+If there are any issues or warnings present in your current project that might result in a build failure or app crash, they will be displayed in this section. You can click this option to view a short description of each issue and navigate to the place where you can fix the issue by clicking on the respective issue.
+
+![Warning-and-errors.avif](imgs/Warning-and-errors.avif)
+
+## 10. Branching
+
+[Branching](#) allows you to create a separate copy of your existing project to build new things without breaking existing features.
+
+## 11. Developer menu
+
+From here, you can view the code for your project, connect to GitHub, and also download the source code for your project.
+
+## 11.1 View Code
+
+This option lets you display the *Dart* code for all the pages of your FlutterFlow project. You can also take a look at the dependencies being used by the app here.
+
+## 11.2 Connect GitHub Repo (Paid Feature)
+
+You can use this option to connect your project with your [GitHub](https://github.com/) account and upload it to a GitHub repository. When you click this option, you will be navigated to the Settings and Integrations > Integrations > Github section.
+
+From there, you can configure your project to add GitHub integration to it. You can get a detailed [step-by-step guide](#) for connecting to GitHub.
+
+### 11.3 Download Code (Paid Feature)
+
+You can download the entire codebase of the app generated by FlutterFlow using this option. It will download a `.zip` file. You can open the contents using any code editor to make modifications.
+
+### 11.4 Download APK (Paid Feature)
+
+Use this to generate a release build of your app. It will automatically download the `.apk` file after the building process is complete.
+
+### 11.5 FlutterFlow CLI
+
+You can also download the code using *[FlutterFlow CLI](https://pub.dev/packages/flutterflow_cli)*. See instructions [here](#).
+
+## 12. Share project
+
```

Comment:
Somewhere here add a link to the Collaboration page from Projects Section

Reply:
Hey @PoojaB26 I think both are different topics. In collaboration, you add users while this section talks about making your project public.

Reply:
Oh okay, sorry, nvm!